### ISSUE TITLE
Feature: "perpetual" roundups

### ISSUE THREAD

**Issue by willhaynes** [INN/link-roundups#18]
I stumbled on [our press page](http://inn.org/about/press/) while looking for something else, and realized that much of what's happening there could be a feature of this plugin.

What if published roundups could be added to after aggregation, so a page like "press" or "industry news" could be consistently updated with a bookmarklet action. Pagination and other features could be built on top.

**Comment by aschweigert** [comment]
yeah, that page is actually powered by the old argo links plugin (just using the link post type but I suppose it could be limited by category or something). i guess the idea of publishing that as a static page or something might be useful though...maybe a shortcode or something that could spit out that query?
